impf/alert.py

```diff
@@ -146,4 +148,19 @@ def pushover_send(message: str) -> None:
 
     r = requests.post(url, data=data)
     if r.status_code != 200: raise AlertError(r.status_code, r.text)
-    logger.debug(r)
\ No newline at end of file
+    logger.debug(r)
+
+@alert_resilience
+def gotify_send(message: str) -> None:
+    url = f'{settings.GOTIFY_URL}/message'
+    params = (
+        ('token', f'{settings.GOTIFY_APP_TOKEN}')
+    )
```

Comment:
Does this work? Never seen tuples used as params, heh. Usually params should be a `dict` – e.g.

```python
params = {
    'token': settings.GOTIFY_APP_TOKEN
}
```

Reply:
Yes, tested with the --alerts argument. works fine!